$OpenBSD: patch-modules_java_CMakeLists_txt,v 1.2 2014/06/03 20:41:36 zhuk Exp $
JNI shared object is installed in the same directory, so this should
be under lib/.
--- modules/java/CMakeLists.txt.orig	Fri Apr 11 12:15:26 2014
+++ modules/java/CMakeLists.txt	Mon May 26 22:11:21 2014
@@ -280,7 +280,7 @@ else(ANDROID)
   if(WIN32)
     set(JAR_INSTALL_DIR java)
   else(WIN32)
-    set(JAR_INSTALL_DIR share/OpenCV/java)
+    set(JAR_INSTALL_DIR lib/OpenCV/java)
   endif(WIN32)
   install(FILES ${JAR_FILE} DESTINATION ${JAR_INSTALL_DIR} COMPONENT java)
 endif(ANDROID)
